sdk/storage/Azure.Storage.Files.Shares/api/Azure.Storage.Files.Shares.netstandard2.0.cs

```diff
@@ -331,8 +331,12 @@ public partial class PermissionInfo
     public partial class ShareAccessPolicy
     {
         public ShareAccessPolicy() { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
Does this change only apply to Shares? I would have thought it would affect Blobs and Queues (and maybe Datalake?) as well.

Reply:
Done. Also Queues does not have this bug (yay!)

Reply:
Out of curiosity, does that mean that the sas will not use queue start/end times if they are default values?

Reply:
Nevermind I see they already nullable. Woohoo!